doc/examples/filters/plot_frangi.py

```diff
@@ -0,0 +1,31 @@
+"""
+==============
+Frangi filter
+==============
+
+Frangi and hybrid Hessian filters can be used for edge detection and
+calculation of fraction of the image containing edges.
```

Comment:
The second part of the sentence after "and" could simply be removed, since they just repeat the first part.